src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { Props } from './types';
+
+
+function Collapsible({
```

Comment:
There should be a comment describing the component just before the Component's declaration.